Todo/JS/PageManager.js

```diff
@@ -0,0 +1,22 @@
+var list = new TaskList();
+document.addEventListener('DOMContentLoaded', function(event) {
+    var button = document.getElementById("new-task-button");
+    button.onclick = generateTask;
+}) 
+
+document.addEventListener('click', event => {

```

Comment:
why not generate event listeners for each button instead of relying on class names?